src/client/components/Overview/Overview.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch, Link } from 'react-router-dom';
+import './Overview.styles.css';
+import Profile from '../../containers/Profile/index';
+// import UsersList from '../UsersListComponent/UsersList';
+import Search from '../Search/Search';
+
+import OverviewSearchMessagesBtn from './OverviewSearchMessagesBtn';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chatTitle">Chats</h3>
+      <Search />
+      {/* <UsersList /> */}
+      <div className="btn-and-profile">
+        <Router>
+          <Link to="/profile">Profile</Link>
+          <Switch>
+            <Route>
+              <Route exact path="/profile">
+                <Profile />
+              </Route>
+              <OverviewSearchMessagesBtn />
+            </Route>
+          </Switch>
+        </Router>
```

Comment:
Malek you keep doing lots of things in this pr that is not part of the PR!

This is why we keep talking about communication and why it is so important. We have spend quite a lot of time on this pr because you misunderstood what you were to do and just started coding and now we are losing time here!

We have a file for the router and that should not be set up here.